contracts/PetPark.sol

```diff
@@ -1,7 +1,108 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+
+    mapping(AnimalType => uint256) private shelter;
+    mapping(address => Person) private borrowers;
+
+    struct Person {
+        uint256 age;
+        Gender gender;
+        AnimalType animal;
+        bool initialized;
```

Comment:
Is this required? Since in solidity storage is very expensive, can you use `age` to check if this has been initialized?

Reply:
nope, not required. makes sense to use age.